schema/schema1.h

```diff
@@ -316,12 +316,12 @@ namespace schema1
 struct ItemLayout: public SPoly<ItemLayout, Layout, 
                                 Join<PositionLayout, VisibilityLayout> >
   {
-    double rotation=0;
-
+	double rotation=0;
     ItemLayout() {}
-    template <class T> ItemLayout(int id, const T& item): 
-      Layout(id), PositionLayout(id, item), VisibilityLayout(item),
-      rotation(item.rotation) {}
+    template <class T> ItemLayout(const T& item): rotation(item.rotation){} 
```

Comment:
There shouldn't be any need to update anything in schema1.

Reply:
I guess once I get the ItemData class working, the problem here will go away.

Reply:
The way the schema classes work is that you can construct a schema n from a schema n-1. That ensures backward compatibility. Finally, for the current schema, you can construct a minsky::Minsky object and construct a current schema object from a minsky::Minsky object. So this means that older schemas (eg schema1) should know nothing about anything in minsky namespace. If they do, it is probably some deprecated code that could and should be removed.